grafana/meta/collectd.yml

```diff
@@ -0,0 +1,10 @@
+{%- if pillar.grafana.server is defined %}
```

Comment:
Are you sure you need this?

Note that collectd will not load grafana's `collectd.yaml` file if there is no `grafana` key in Pillar. See https://github.com/tcpcloud/salt-formula-collectd/blob/master/collectd/client.sls#L59.